src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,9 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ arrOfComments }) => (
+  <div className="CommentList">
```

Comment:
use semantic tags
fix all similar cases

Reply:
https://github.com/mate-academy/react_static-list-of-posts-js/pull/85/files#r1257455608
not fixed. better to use semantid tags. `ul` in this case
ans `li` in CommentInfo.
pay attention on it in next tasks